meta-phosphor/common/recipes-phosphor/obmc-phosphor-event/files/obmc-phosphor-event.service

```diff
@@ -2,7 +2,7 @@
 Description=Phosphor OpenBMC event management daemon
 
 [Service]
-ExecStart=/usr/sbin/obmc-phosphor-eventd
+ExecStart=/usr/sbin/obmc-phosphor-eventd -s 200000
```

Comment:
IMO we should use EnvironmentFile entry under the service section and read the option from that file instead of hardcoding the value directly. This option will even help to change the value runtime and test it.